woodwork/statistics_utils/_determine_best_outlier_method.py

```diff
@@ -0,0 +1,66 @@
+from collections import namedtuple
+
+import numpy as np
+import pandas as pd
+
+import woodwork as ww
+from woodwork.statistics_utils._get_medcouple_statistic import _get_medcouple
+
+
+def _sample_for_medcouple(series, seed=42):
+    """Samples the data in order to calculate the medcouple statistic.
+
+    Args:
+        series (Series): Data on which sampling will occur.
+        seed (int): Seed for random sampling.
+
+    Note:
+        The calculation of the medcouple statistic has a large memory requirement of O(N**2), therefore a series over 10,000
+        observations will be randomly sampled up to 10,000.
+
+    Returns:
+        float: Medcouple statistic from the sampled data
+    """
+    np.random.seed(seed)
+    series_size = len(series)
+    if series_size < ww.config.get_option("medcouple_sample_size"):
+        series_sample = series.copy()
+    else:
+        series_sample = np.random.choice(
```

Comment:
does this include the seed?

Reply:
The seed is initialized a few lines above `np.random.seed(seed)`